app/assets/stylesheets/modules/results.scss

```diff
@@ -31,6 +31,13 @@
   }
 }
 
+.description {
+  -webkit-box-orient: vertical;
+  display: -webkit-box;
+  -webkit-line-clamp: 2;
```

Comment:
@jkeck `webkit-line-clamp` is looking good in Firefox, Safari, and Chrome. What are our options for testing this out in Internet Explorer? 
-webkit-line-clamp is not supported in Internet Explorer. So I _think_ this might be why @mejackreed  needed to use [line-clamp stuff mylibrary](https://github.com/sul-dlss/mylibrary/blob/c67a52db65bb6088dd05d1513fb82041ca19d033/app/assets/stylesheets/mixins.scss#L26-L33).
### Notes 
- [webkit-line-clamp MDN ](https://developer.mozilla.org/en-US/docs/Web/CSS/-webkit-line-clamp)
- [line-clamp](https://caniuse.com/#feat=css-line-clamp)

Reply:
No prob, I've taken the same approach that we have in MyLibrary.